src/HW_10/task 10_1.py

```diff
@@ -0,0 +1,47 @@
+"""Создать csv файл с данными следующей структуры: Имя, Фамилия,
+Возраст. Создать отчетный файл с информацией по количеству людей
+входящих в ту или иную возрастную группу. Возрастные группы: 1-12, 13-18,
+19-25, 26-40, 40+."""
+
+import csv
+
+group_of_ages = [
+    ['1-12'],
+    ['13-18'],
+    ['19-25'],
+    ['26-40'],
+    ['40+']
+]
+
+def print_csv_file(filename):
+    with open(f"{filename}.csv") as my_file:
+        for line in my_file.readlines():
+            print(line)
+
+def csv_writer(final_file, name_of_file):
+    with open(name_of_file, "w", newline='') as csv_file:
+        writer = csv.writer(csv_file, delimiter=',')
+        for line in final_file:
+            writer.writerow(line)
+
+def check_group_of_ages(filename, final_file):
```

Comment:
```suggestion
def check_group_of_ages(filename: str, final_file):
```

Please add docstring to each method